tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/rules/navigate-to-rules.when.js

```diff
@@ -3,12 +3,12 @@ import { clickElement, getObject } from '../../../utils/driver';
 import {
   managementButton,
   wazuhMenuButton,
-  rulesButton,
+  rulesLink,
 } from '../../../pageobjects/wazuh-menu/wazuh-menu.page';
 
 When('The user navigates to rules', () => {
   clickElement(wazuhMenuButton);
   clickElement(managementButton);
-  clickElement(rulesButton);
+  clickElement(rulesLink);
   cy.wait(3000);
```

Comment:
This `cy.wait()` its neccessary?

Reply:
it is not, but I already deleted all of them, and merged to `feature/frontend`

Maybe, he needs to pull `feature/frontend` into his branch

Reply:
Yes, as @MauGaP mentioned, I needed to update my branch.